org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/AnyType.xtend

```diff
@@ -14,7 +14,7 @@ class AnyType extends BasicType implements WollokType {
 		previouslyInferred
 	}
 	
-	override acceptAssignment(WollokType other) {}
+	override acceptsAssignment(WollokType other) { true }
```

Comment:
I added #acceptsAssignment which returns a boolean instead of throwing an exception. Unless you want to customize the error thrown, it is easier to override the new method, as the older one now delegates to it.